components/perl/Mail-SPF/perl-integrate-module.conf

```diff
@@ -0,0 +1,34 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2022 Friedrich Kink
+#
+
+%hook-no-license%
+printf "%s\n" "$COPYRIGHT" > "$DISTRIBUTION.license"
+LICENSE="MPL-1.1"
+USE_DEFAULT_PERL_LICENSE=0
+
+%include-1%
+# Force modulebuild build style
+BUILD_STYLE = modulebuild
+
+%include-2%
+COMPONENT_SRC =                 $(COMPONENT_NAME)-v$(HUMAN_VERSION)
+
+%include-3%
+# see https://rt.cpan.org/Ticket/Display.html?id=93241
+COMPONENT_PREP_ACTION =+ rm -f $(SOURCE_DIR)/t/90-author-pod-validation.t
+
+%hook-manifest%
+# this daemon is not needed for the package
+/usr/gnu/bin/sed -i -e '/spfd$/d' "$DISTRIBUTION-PERLVER.p5m"
```

Comment:
I suggest to add something like:

```
GENERATE_EXTRA_CMD += | grep -v spfd
```

into `%include-3%`.  This is far simpler and it will also remove the file from sample manifest.

Reply:
ok